### DISCUSSION TITLE
Can't install brew on macOS big sur

### DISCUSSION THREAD

**Discussion by datlechin** [Homebrew/discussions#1132] [Everyday usage]
![Screen Shot 2021-03-26 at 01 50 57](https://user-images.githubusercontent.com/56961917/112606848-c0f89880-8dd5-11eb-815c-4b6be1d37a0a.png)

```
==> /usr/bin/sudo /bin/mkdir -p /usr/local/bin
mkdir: /usr/local/bin: Not a directory
Failed during: /usr/bin/sudo /bin/mkdir -p /usr/local/bin
```

Help me

**Comment by SMillerDev** [comment]
What does `ls -al /usr/` say? And what about `ls -al /usr/local/`?

**Reply by datlechin** [reply]
![Screen Shot 2021-03-26 at 02 08 20](https://user-images.githubusercontent.com/56961917/112608920-251c5c00-8dd8-11eb-807d-865294de8143.png)
![Screen Shot 2021-03-26 at 02 08 44](https://user-images.githubusercontent.com/56961917/112608967-336a7800-8dd8-11eb-93a3-6c1352d87494.png)

**Reply by SMillerDev** [reply]
Please don't post output as screenshots but just copy the text. Images make the whole thing a lot more difficult. 

The issue is that somehow you already have a `/usr/local/bin` on your system and it's a file. You can remove that file (`rm /usr/local/bin`) and run the installer again to continue.

**Reply by datlechin** [reply]
Thank youuuuu
